src/api/call.ts

```diff
@@ -1,10 +1,11 @@
 // SPDX-License-Identifier: Apache-2.0
 "use strict";
 
-import { ErdstallObject } from "./object";
+import { ErdstallObject } from "#erdstall/api/object";
```

Comment:
I propose that _within_ a directory, we keep relative imports. This has the advantage that if we were to rename or move the directory they reside in, we don't have to change those imports. This is also more similar to how we can currently do dir-local imports in Go :) In particular, all `export * from './foo'` in `index.ts` files should also be relative.

Reply:
Agree.